sm_cnn/README.md

```diff
@@ -1,80 +1,149 @@
 ## SM model
 
 #### References:
-1. Aliaksei _S_everyn and Alessandro _M_oschitti. 2015. Learning to Rank Short Text Pairs with Convolutional Deep Neural Networks. In Proceedings of the 38th International ACM SIGIR Conference on Research and Development in Information Retrieval (SIGIR '15). ACM, New York, NY, USA, 373-382. DOI: http://dx.doi.org/10.1145/2766462.2767738
+1. Aliaksei _S_everyn and Alessandro _M_oschitti. 2015. Learning to Rank Short Text Pairs with Convolutional Deep Neural 
+Networks. In Proceedings of the 38th International ACM SIGIR Conference on Research and Development in Information 
+Retrieval (SIGIR '15). ACM, New York, NY, USA, 373-382. DOI: http://dx.doi.org/10.1145/2766462.2767738
 
 
 ### Requirements
-gensim==1.0.1
+```
 nltk==3.2.2
 numpy==1.11.3
-pandas==0.19.2
-torch==0.1.11+b13b701
+pytorch==0.1.12
+gensim==1.0.1 
+```
 
-Please install the requirements. See [Castor/README.md](../README.md) for pytorch installation details.
+The code uses torchtext for text processing. Set torchtext:
+```bash
+git clone https://github.com/pytorch/text.git
+cd text
 
-### Training the model
+#use this commit number
+git reset --hard 2980f1bc39ba6af332c5c2783da8bee109796d4c
+python setup.py install
+```
 
-1. Setup repository layout.
+We use `trec_eval` for evaluation:
 
+```bash
+cd eval
```

Comment:
I suggest we consolidate all trec_eval utils - check out this utility function for getting `map` and `mrr`: https://github.com/castorini/Castor/blob/master/utils/relevancy_metrics.py#L6. Check here for a sample usage: https://github.com/castorini/Castor/blob/61c8c0e622caa69e8579174dee429901128dc7e1/nce/NCE-Pairwise-SM/main.py#L73.